src/components/adherence/summary/AdherenceAlerts.test.tsx

```diff
@@ -0,0 +1,85 @@
+import * as useUserSessionDataState from '@helpers/AuthContext'
+import {render, screen, waitFor} from '@testing-library/react'
+import constants from '@typedefs/constants'
+import {rest} from 'msw'
+import {QueryClient, QueryClientProvider} from 'react-query'
+import {MemoryRouter} from 'react-router-dom'
+import {loggedInSessionData} from '__test_utils/mocks/user'
+import server from '__test_utils/test_server/server'
+import {ProvideTheme} from '__test_utils/utils'
+import AdherenceAlerts from './AdherenceAlerts'
+
+const studyId = 'hprczm'
+
+jest.mock('@helpers/AuthContext')
+
+const mockedAuth = useUserSessionDataState as jest.Mocked<typeof useUserSessionDataState>
+mockedAuth.useUserSessionDataState.mockImplementation(() => loggedInSessionData)
+
+export const renderComponent = () => {
+  const userData = loggedInSessionData
+  mockedAuth.useUserSessionDataState.mockImplementation(() => userData)
+
+  const queryClient = new QueryClient()
+
+  const element = render(
```

Comment:
In [util.tsx](https://github.com/Sage-Bionetworks/mtb/pull/708/files#diff-16fe1543612af3aa72a8c1c9d85e0cefe8a2e54482d61d8a7b901aedababebbeR1) there is a Wrapper function that does a lot of that. I think it would be a fun experiment to make it configurable that we can reuse it by providing it the wrappers we need